### ISSUE TITLE
Failure within Google Colab Python Notebook environment

### ISSUE THREAD

**Issue by thoughtful-solutions** [tristansaminadayar/pandas-datacube#1]
When making use of [Google Colab](https://colab.research.google.com/)
it is possible to successfully install the pandas-datacube package.
```
!pip install pandas-datacube
```

However when attempting to import the package 

```
from pandasdatacube import get_datasets
import pandas as pd
```

it fails with the following errors

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-c9ef2fe4db74> in <module>
----> 1 from pandasdatacube import get_datasets
      2 import pandas as pd
      3 
      4 

1 frames
/usr/local/lib/python3.8/dist-packages/pandasdatacube/pandasdatacube.py in <module>
      8 
      9 
---> 10 def expand_name(word: str, prefixes: dict[str]) -> str:
     11     """
     12     Function that transforms the prefix using a dictionary of conversions

TypeError: 'type' object is not subscriptable
```

This error doesn't appear to occur when using python natively within a VM or desktop environment

**Comment by serasset** [comment]
It may be due to python being < 3.9 

What is the python version of your Collab instance ? Is it possible to use a ≥ 3.9 python in your Collab Notebook ?

**Comment by serasset** [comment]
Fixed in 0.0.4 AFAICT
